apps/src/aichat/views/ChatWarningModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
```

Comment:
Super nit: remove newline?

Reply:
That's intended, to visually separate lib imports from 'internal' project imports. I can remove it if there's a rule for this, but I've used to do it pretty often, code-dot-org repo included :D